src/EnergyPlus/VentilatedSlab.cc

```diff
@@ -131,37 +131,13 @@ namespace VentilatedSlab {
     static std::string const fluidNameWater("WATER");
     std::string const cMO_VentilatedSlab = "ZoneHVAC:VentilatedSlab";
 
```

Comment:
If these things were only defined in the `.cc` file you can also move the `enum class` definitions into the `.cc` file too, there is nothing that says they have to be in the `.hh` file. In fact ,the less we have in `.hh` files the better.

Reply:
Will remember for future cases. But in this case, when I converted the `int constexpr` to `enum class`, I also converted the type of variables that used them to that enum type. And these variables are defined in the `.hh` file.